File: src/main/java/random/telegramhomebot/config/CommandsConfig.java
package random.telegramhomebot.config;

import org.springframework.beans.factory.config.PropertiesFactoryBean;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import org.springframework.core.io.ClassPathResource;

@Configuration
public class CommandsConfig {
	@Bean(name = "telegramCommands")
	public PropertiesFactoryBean telegramCommands() {
		PropertiesFactoryBean bean = new PropertiesFactoryBean();
		bean.setLocation(new ClassPathResource("commands.properties"));
		return bean;
	}
}

File: src/main/java/random/telegramhomebot/config/CommandsConfig.kt
package random.telegramhomebot.config

import org.springframework.beans.factory.config.PropertiesFactoryBean
import org.springframework.context.annotation.Bean
import org.springframework.context.annotation.Configuration
import org.springframework.core.io.ClassPathResource

@Configuration
class CommandsConfig {
    @Bean(name = ["telegramCommands"])
    fun telegramCommands(): PropertiesFactoryBean {
        val bean = PropertiesFactoryBean()
        bean.setLocation(ClassPathResource("commands.properties"))
        return bean
    }
}